### DISCUSSION TITLE
Bticino livinglight smart Fastidioso ronzio

### DISCUSSION THREAD

**Discussion by Gianlucap84** [indomus/forum#225] [Hardware & Firmware]
Ciao a tutti, ho una domanda per chi ha installato la serie civile bticino livinglight smart utlizzando i moduli connessi gateway, tapparelle, deviatori etc zigbee. Nello specifico Deviatori N4003C Dimmer N4411C Comandi tapparella N4027C. Il quesito è se anche voi avvertite un ronzio fastidioso da tutti i moduli a prescindere se sono in funzione o meno(es. mentre la tapparella sale o scende). I miei hanno tutti, tranne il gateway questo ronzio che di giorno non si percepisce mentre di notte il fastidioso fischio si sente di più. Qualcuno ha lo stesso problema?

**Comment by essegi65** [comment]
Io ho la serie Living Now e questa serie non produce nessun ronzio, ne il gateway ne i vari frutti (deviatori connessi o no, prese, ecc.)
